src/modules/new-request-form/answer-bot-generative-modal/GenerativeAnswerBotModal.tsx

```diff
@@ -83,17 +91,33 @@ export function GenerativeAnswerBotModal({
     fetchGeneratedAnswer();
   }, [requestId]);
 
-  const redirectToHelpCenterPath = () => (window.location.href = redirectTo);
+  const getRedirectUrl = () => {
+    if (!isSignedIn) {
+      const searchParams = new URLSearchParams();
+      searchParams.set("return_to", requestsPath);
+      return `${helpCenterPath}?${searchParams.toString()}`;
+    } else if (hasRequestManagement) {
```

Comment:
do we need this conditional branch? as long as the user is signed in they can reach the requests list page, or is this leading somewhere else?